etl/config.py

```diff
@@ -123,9 +123,15 @@ def variable_metadata_url(variable_id):
 # any garden or grapher dataset after this date will have strict mode enabled
 STRICT_AFTER = "2023-06-25"
 
+SLACK_API_TOKEN = env.get("SLACK_API_TOKEN")
+
+# if True, commit and push updates to YAML files coming from admin
+ETL_API_COMMIT = env.get("ETL_API_COMMIT") in ("True", "true", "1")
```

Comment:
Note: an alternative to checking for all these truthy values is just to do the behaviour if the key is present at all.